src/Entity/Package.php

```diff
@@ -539,6 +542,14 @@ public function getBrowsableRepository(): string
     public function addVersion(Version $version): void
     {
         $this->versions[] = $version;
+
+        $newVersion = strtolower($version->getNormalizedVersion());
+
+        if (null === $this->snapshotedLatestVersion
+            ||
+            version_compare($newVersion, $this->snapshotedLatestVersion, '>=')) {
```

Comment:
`version_compare` looks wrong to me here given that you don't only include the version when writing in the property.

Reply:
indeed it was my original idea, 2 fields, but for snapshots data I went for one
i handle here the explode